lib/beaker/subcommands/subcommand_util.rb

```diff
@@ -16,18 +16,33 @@ module SubcommandUtil
       SUBCOMMAND_STATE = Pathname("#{CONFIG_DIR}/.subcommand_state.yaml")
       PERSISTED_HOSTS = Pathname("#{CONFIG_DIR}/.hosts.yaml")
       PERSISTED_HYPERVISORS = Pathname("#{CONFIG_DIR}/.hypervisors.yaml")
+      # These options should not be part of persisted subcommand state
+      UNPERSISTED_OPTIONS = [:beaker_version, :command_line, :hosts_file, :logger, :password_prompt, :timestamp, ]
```

Comment:
It looks like `:password_prompt` is a new option compared to the old array we were removing. Was this intentional?

Reply:
Ah, yes -- I'll update the commit message to make that clear. This is an object I saw being erroneously persisted as a string in beaker-docker.